internal/gensupport/send.go

```diff
@@ -99,6 +99,19 @@ func sendAndRetry(ctx context.Context, client *http.Client, req *http.Request, r
 		case <-time.After(pause):
 		}
 
+		select {
```

Comment:
Instead of a selecting I think we can just check if ctx.Err() is non-nil.

Reply:
yup, we totally can. TIL that `ctx.Err()` is threadsafe.